cardano_node_tests/tests/tests_conway/test_committee.py

```diff
@@ -139,87 +129,18 @@ def test_register_and_resign_committee_member(
-        [r.success() for r in (reqc.cli032, reqc.cip002, reqc.cip004)]
-
-        # Resignation of CC Member
-
-        _url = helpers.get_vcs_link()
-        [r.start(url=_url) for r in (reqc.cli007, reqc.cip012)]
-        res_cert = cluster.g_conway_governance.committee.gen_cold_key_resignation_cert(
-            key_name=temp_template,
-            cold_vkey_file=cc_auth_record.cold_key_pair.vkey_file,
-            resignation_metadata_url="http://www.cc-resign.com",
-            resignation_metadata_hash="5d372dca1a4cc90d7d16d966c48270e33e3aa0abcb0e78f0d5ca7ff330d2245d",
-        )
-        reqc.cli007.success()
-
-        tx_files_res = clusterlib.TxFiles(
-            certificate_files=[res_cert],
-            signing_key_files=[payment_addr_comm.skey_file, cc_auth_record.cold_key_pair.skey_file],
-        )
-
-        tx_output_res = clusterlib_utils.build_and_submit_tx(
-            cluster_obj=cluster,
-            name_template=f"{temp_template}_res",
-            src_address=payment_addr_comm.address,
-            submit_method=submit_method,
-            use_build_cmd=use_build_cmd,
-            tx_files=tx_files_res,
-        )
-
-        cluster.wait_for_new_block(new_blocks=2)
-        res_committee_state = cluster.g_conway_governance.query.committee_state()
-        res_member_rec = res_committee_state["committee"].get(member_key)
-        assert (
-            not res_member_rec or res_member_rec["hotCredsAuthStatus"]["tag"] == "MemberResigned"
-        ), "CC Member not resigned"
-        reqc.cip012.success()
-
-        res_out_utxos = cluster.g_query.get_utxo(tx_raw_output=tx_output_res)
-        assert (
-            clusterlib.filter_utxos(utxos=res_out_utxos, address=payment_addr_comm.address)[
-                0
-            ].amount
-            == clusterlib.calculate_utxos_balance(tx_output_res.txins) - tx_output_res.fee
-        ), f"Incorrect balance for source address `{payment_addr_comm.address}`"
-
-        # Check CC member in db-sync
-        dbsync_utils.check_committee_member_registration(
-            cc_member_cold_key=cc_auth_record.key_hash, committee_state=auth_committee_state
-        )
-        dbsync_utils.check_committee_member_deregistration(
-            cc_member_cold_key=cc_auth_record.key_hash
-        )
-        cluster.wait_for_new_epoch()
+        # Try to submit a Hot Credential Authorization certificate without being a CC member
+        with pytest.raises(clusterlib.CLIError) as excinfo:
+            clusterlib_utils.build_and_submit_tx(
+                cluster_obj=cluster,
+                name_template=f"{temp_template}_auth",
+                src_address=payment_addr_comm.address,
+                submit_method=submit_method,
+                use_build_cmd=use_build_cmd,
+                tx_files=tx_files_auth,
+            )
+        err_str = str(excinfo.value)
+        assert "ConwayCommitteeIsUnknown" in err_str, err_str
```

Comment:
This changes existing test to a test with different scope.
We still need to test CC members registration and resignation and the linked user stories, even though the behavior changed.

Reply:
Register a CC member is already present in other tests, maybe we can refactor the test just for the resignation? And add the test to check the error?